examples/00_quick_start/als_movielens.ipynb

```diff
@@ -2,46 +2,34 @@
  "cells": [
```

Comment:
<p>@anargyri, queston about this, why are you checking if we are on databricks? </p><p>are you trying to make sure that the exact same notebook can work on a DSVM and in Databricks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1531/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Yes, now the notebook won't fail on Databricks (it was failing because `start_or_get_spark()` fails).
This was a suggestion by Le here https://github.com/microsoft/recommenders/issues/1439

Reply:
one thing that is a little bit weird is that we have `start_or_get_spark` but then we are adding an if clause to all notebooks. 

in the code of that function at the end we have:
```
spark_opts.append("getOrCreate()")
```
so if we add `start_or_get_spark` into Databricks, since spark instance already exists, it should return it directly.

Based on that, it should be possible to remove the if in:
```
if not is_databricks():
    spark = start_or_get_spark("ALS PySpark", memory="16g")
```
and leave only:
```
spark = start_or_get_spark("ALS PySpark", memory="16g")
```

Reply:
Actually, if you call `start_or_get_spark()` on Databricks it fails.
We already had the `if not is_databricks():` clause before this PR in a couple of notebooks. I just added it to the rest.

Reply:
But one alternative that may look better is to incorporate the `if not is_databricks()` check inside the definition of `start_or_get_spark()`.

Reply:
I think this makes sense, if we put the if not is_databricks inside start_or_get_spark we can raise a warning that that function is not supported on spark. that said, do we know why it's failing and would it be better to resolve that?

Reply:
Hmm, I just tried `start_or_get_spark()` on ADB with the PyPI reocmmenders package and it works.
I am pretty sure it failed at some point, but now I wonder if it was failing because of something wrong in the installation process.

Reply:
I think what I was doing was something like
```
%%bash
python -c "from reco_utils.common.spark_utils import start_or_get_spark
start_or_get_spark()"
```
inside a cell. This raises an error in `start_or_get_spark()`. 
I doubt though that this is a recommended way to use Databricks.

Reply:
FYI @yueguoguo in case you have more experience with `start_or_get_spark()` on Databricks.

Reply:
> I just tried start_or_get_spark() on ADB with the PyPI recommenders package and it works

ok this is cool, and I guess the same function also works on DSVM spark right? then there is no need to add the if clause

Reply:
Right. I can remove the `if not is_databricks()`, except where it is needed for other reasons.